models/user.js

```diff
@@ -1,18 +1,64 @@
-// copied and pasted from activity 15, in case there is confusion. I did change the names
 module.exports = function(sequelize, DataTypes) {
-  var User = sequelize.define("user", {
-    // Giving the User model a name of type STRING
-    name: DataTypes.STRING
-    // Add more columns as needed
-  });
+var User = sequelize.define("User", {
+  id: {
+    type: DataTypes.UUID,
+    primaryKey: true,
+    defaultValue: DataTypes.UUIDV4, //set to REFERENCE the Universal-Unique-ID est above... 
+    allowNull: false
+  },
+  name: {
+    type: DataTypes.STRING,
+    required: true
+  },
+  gender: {
+    type: DataTypes.ENUM,
+    values: ["Woman", "Man", "TransWoman", "TransMan"],
+    allowNull: false
+  },
+  age: {
+    type: DataTypes.INTEGER,
+    allowNull: false
+  },
+  phoneNumber: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  genderPreference: {
```

Comment:
same as the gender. also be care full about changing variable names. I have already established these column names